### DISCUSSION TITLE
Problem with "Store variables in local storage"

### DISCUSSION THREAD

**Discussion by piw-piw** [TurboWarp/packager#30] [Q&A] [answered]
I have a problem with "Store variables in local memory", it doesn't work.
I open the file, save variable, close file,
And when I open file again, variable hasn't changed.

**Comment by GarboMuffin** [comment, marked as answer]
What browser/environment?

**Reply by piw-piw** [reply]
Chrome

**Reply by GarboMuffin** [reply]
Something with the browser is making it not save data, then. I can't do anything about that. It should work if you put it on a website or in a proper packaged environment.

**Reply by GarboMuffin** [reply]
You were probably affected by #38, which is now fixed.
